h2o-core/src/main/java/hex/Model.java

```diff
@@ -1731,7 +1731,12 @@ else if (isWeights && computeMetrics) {
           if (expensive) {
             vec = test.anyVec().makeCon(1);
             toDelete.put(vec._key, "adapted missing vectors");
-            msgs.add(H2O.technote(1, "Test/Validation dataset is missing weights column '" + names[i] + "' (needed because a response was found and metrics are to be computed): substituting in a column of 1s"));
+            // cross-validation generated weights will not be found in test/validation dataset.  This warning is
+            // invalid.  We will suppress this warning.
+            if (!names[i].contains("_internal_cv_weights_"))
```

Comment:
nit: pls wrap  the command into a curly braces `{}` (just to avoid confusions)

Reply:
Done